getCalspec/rebuild.py

```diff
@@ -43,9 +45,24 @@ def add_astroquery_id(df):
 
 
 def add_alt_star_name(df):
+    name_columns = [name for name in df.columns if "name" in name.lower()]
+    print(name_columns)
     for i, row in df.iterrows():
         if row["Star name"] == "ETA1 DOR":
             df.at[i, "Alt Star name"] = "ETA DOR"
+        else:
+            all_names = None
+            for name in name_columns:
+                all_names = Simbad.query_objectids(row[name])
+                if all_names is not None and len(all_names) > 0:
+                    break
+            if all_names is not None:
+                for name in list(all_names['ID']):
+                    if "HD" in name:
+                        df.at[i, "Alt Star name"] = name.replace(' ','')
```

Comment:
I guess you don't need the `.upper()` call here because they're all already upper case here? But again, if you had a utility function they you wouldn't need to worry about consistency. I assume there's no stars that aren't HD stars which contain the string `HD` by coincidence then? Might be safer to do `name.startswith('HD')` anyway though?

Reply:
OK, I will even add a 'HD_name' in calspec.csv.